python/cudf_polars/cudf_polars/dsl/utils/aggregations.py

```diff
@@ -125,6 +125,24 @@ def decompose_single_agg(
         )
         if any(has_agg for _, has_agg in aggs):
             raise NotImplementedError("Nested aggs in groupby not supported")
+
+        child_dtype = child.dtype.plc
+        req = agg.agg_request
+        is_median = agg.name == "median"
+        is_quantile = agg.name == "quantile"
+
+        is_group_quantile_supported = plc.traits.is_integral(
+            child_dtype
+        ) or plc.traits.is_floating_point(child_dtype)
```

Comment:
Question: Confirming that this case isn't covered by `plc.aggregation.is_valid_aggregation`?

Reply:
It wasn't. I think what I would like is an API like `is_valid_aggregation(dtype, agg, context)` where context is an enum like `GROUPBY`, `ROLLING`. I think some aggregations are supported in certain dtypes an not in others, so this API would help distinguish

Reply:
Started writing up thoughts on this in https://github.com/rapidsai/cudf/issues/19699